app/client/components/helpCentre/helpCentreNav.tsx

```diff
@@ -0,0 +1,144 @@
+import { css } from "@emotion/core";
+import { space } from "@guardian/src-foundations";
+import { neutral } from "@guardian/src-foundations/palette";
+import { textSans } from "@guardian/src-foundations/typography";
+import { Link } from "@reach/router";
+import React, { useState } from "react";
+import { maxWidth, minWidth } from "../../styles/breakpoints";
+import { HelpCentreNavConfig, helpCentreNavConfig } from "./helpCentreConfig";
+import {
+  containterCss,
+  innerSectionCss,
+  innerSectionDivCss,
+  linkAnchorStyle,
+  linkArrowStyle,
+  sectionTitleCss
+} from "./helpCentreStyles";
+
+interface HelpCentreNavProps {
+  selectedTopicObject?: HelpCentreNavConfig;
+}
+
+const ulCss = css`
+  list-style: none;
+  margin: 0 0 ${space[6]}px 0;
+  padding: 0;
+  ${maxWidth.desktop} {
+    display: none;
+  }
+`;
+
+const liCss = (isSelectedTopic: boolean, isFirstTopic: boolean) => css`
+  ${textSans.medium()};
+  color: ${neutral["7"]};
+  border-left: ${isSelectedTopic
+    ? `${space[2]}px solid #121212`
+    : `${space[2]}px solid #dcdcdc`};
+  font-weight: ${isSelectedTopic ? "700" : "normal"};
+  cursor: pointer;
+  ::after {
+    content: "";
+    display: block;
+    border-bottom: 1px solid ${neutral["86"]};
+  }
+  ::before {
+    content: "";
+    display: ${isFirstTopic ? "block" : "none"};
+    border-top: 1px solid ${neutral["86"]};
+  }
+`;
+
+const pCss = css`
+  padding: ${space[4]}px ${space[3]}px;
+  margin: 0;
+`;
+
+const HelpCentreNav = (props: HelpCentreNavProps) => {
+  const [open, setOpen] = useState<boolean>(false);
+
+  const handleSectionClick = () => () => {
+    setOpen(!open);
+  };
```

Comment:
This will also work. (together with the suggestion on line 98)
```suggestion
  const handleSectionClick = () => setOpen(!open);
 ```